ENH: auto-profile module execution (#323)
pyutils/line_profiler: CHANGELOG.rst

* ENH: Add support for ``sys.monitoring`` (Python >= 3.12)
* FIX: Fixed issue when calling ``kernprof`` with neither the ``-l`` nor ``-b`` flag; also refactored common methods to ``LineProfiler`` and ``ContextualProfile``
* FIX: Fixed auto-profiling of async function definitions #330

> * ENH: Added CLI argument ``-m`` to ``kernprof`` for (auto-)profiling module/package execution instead of that of scripts.

Reviewer: ```suggestion
* ENH: Added CLI argument ``-m`` to ``kernprof`` for running a library module as a script```
